project/Dependencies.scala

```diff
@@ -14,13 +14,15 @@ object Dependencies {
 
   val ClusterHttp = Seq(
     libraryDependencies ++= Seq(
-      "com.typesafe.akka" %% "akka-cluster"                      % AkkaVersion,
-      "com.typesafe.akka" %% "akka-http"                         % AkkaHttpVersion,
-      "com.typesafe.akka" %% "akka-http-spray-json"              % AkkaHttpVersion,
-      "io.spray"          %% "spray-json"                        % "1.3.2",                  // ApacheV2
-      "com.typesafe.akka" %% "akka-http-testkit"                 % AkkaHttpVersion % "test",
-      "junit"             % "junit"                              % junitVersion    % "test",
-      "org.mockito"       % "mockito-all"                        % "1.10.19"       % "test"  // Common Public License 1.0
+      "com.typesafe.akka" %% "akka-cluster"                       % AkkaVersion,
+      "com.typesafe.akka" %% "akka-cluster-sharding"              % AkkaVersion    % "provided",
```

Comment:
remove "provided", as implemented now the dependency is required and I think we agreed that would be fine